cranelift/filetests/filetests/isa/s390x/jumptable.clif

```diff
@@ -32,7 +32,7 @@ block5(v5: i32):
 ;   clgfi %r3, 3
 ;   jghe label4
 ;   sllg %r3, %r3, 2
-;   larl %r1, 14 ; agf %r1, 0(%r1, %r3) ; br %r1 ; jt_entries label3 label2 label1
+;   larl %r1, 14 ; agf %r1, 0(%r1, %r3) ; br %r1 ; jt_entries label2 label1
```

Comment:
What happened to label3?

Reply:
Previously all jump targets, plus the default, was stored in a list where the first element of the list was implicitly the default. Now they're stored separately, the default in one location and the targets in another. This display impl I left rendering only the targets as those are the jump table entries.